lib/utils.re

```diff
@@ -60,6 +60,24 @@ let parseStylesheet = (~containerLnum=?, ~pos=?, css) =>
     )
   };
 
+/* Get all the classes from a given selector (prelude) */
+let getClassesFromSelector = selector => {
+    let rec get_classes = (classes, selector) => {
```

Comment:
Just curious, but why not camelCase (`getClasses`) here? If there's a specific reason that's totally fine, but otherwise I'd love to keep the naming conventions in the code consistent.

Reply:
Oh, I slipped, wrote the names on snake case then rewrote as camel case, will push fix in a bit